fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2022,8 +2027,13 @@ void setupSimulatedSystem(vector<Future<Void>>* systemActors,
 					processClass = ProcessClass((ProcessClass::ClassType)deterministicRandom()->randomInt(0, 3),
 					                            ProcessClass::CommandLineSource); // Unset, Storage, or Transaction
 				if (processClass ==
-				    ProcessClass::StatelessClass) // *can't* be assigned to other roles, even in an emergency
+				    ProcessClass::StatelessClass) { // *can't* be assigned to other roles, even in an emergency
 					nonVersatileMachines++;
+					possible_ss--;
+				}
```

Comment:
There are a ton of process class types, wouldn't it be easier (and also correct for weird types we don't use in simulation currently like DataDistributorClass) to just set this to the number of UnsetClass + StorageClass?